src/oop/programs_with_multiple_classes/hard/q2/QuestionText.md

```diff
@@ -0,0 +1,39 @@
+# oop-programs_with_multiple_classes-hard-q2
+
+You want to write a program that models a group of people and their blanket fort. To do this, create a Java program with
+three classes: `Person`, `Blanket`, and `TestFort`. See the requirements below for each class. Note that a `toString`
+method has been provided in the `Person` and `Blanket` classes for you to use.
+
+## Person Class
+
+Your `Person` class should have the following items:
+
+- Attributes:
+    - A `double` called **height**
+- Constructors:
+    - A `public Person(double)` constructor which initializes the **height** field with the passed double
```

Comment:
For this, I think Bowen wants us to present constructors and methods differently. Might want to say something more like "A constructor which accepts a double to initialize the height field and is accessible from other classes. "

Reply:
Instead of just outright giving them the method header.